reports/ssh_report.txt

```diff
@@ -0,0 +1 @@
+RESULT fail security.passwords.ssh Default passwords have not been changed.
```

Comment:
??

Reply:
This was made by accident when testing the code, thanks for spotting this. Will also remove this!